src/Client/Client.ts

```diff
@@ -51,4 +51,9 @@ export interface Client extends TypedEmitter<ClientEvents> {
 	 * @param port - port number of daemon on the host
 	 */
 	introduce(host: string, port: number): void
+
+	/**
+	 * Close all daemons
+	 */
+	closeAllDaemonsAndFinish(): void
```

Comment:
Could you rename this method to be shorter in both classes? E.g. `quit`.

Reply:
Will do.